### ISSUE TITLE
Setup CI for builds & NuGet packaging

### ISSUE THREAD

**Issue by CPonty** [CPonty/templ-dot-net#7]
Potential resources:
- http://docs.myget.org/docs/reference/build-services
- http://www.appveyor.com/docs/status-badges

**Comment by ichpuchtli** [comment]
continuous myget builds would be sweet.

**Comment by ichpuchtli** [comment]
shields.io is a free badge server which scrapes web pages there one for nuget showing downloads however i don't think it looks at the right number in nuget. 

![](https://img.shields.io/nuget/dt/Templ.net.svg)

**Comment by CPonty** [comment]
Yeah unfortunately it seems to be looking at the numbers from the first version of the package only. Will leave it off, for now

**Comment by CPonty** [comment]
Added: <img alt="NugetVersion" src="https://img.shields.io/nuget/v/Templ.NET.svg" />
